src/core.c

```diff
@@ -3540,6 +3540,17 @@ Vector2 GetMousePosition(void)
     return position;
 }
 
+// Mouse movement per frame XY
+Vector2 GetMouseDelta()
```

Comment:
Functions in C require the input parameter definition: `GetMouseDelta(void)`, your declaration implies an indeterminate number of input parameters.